corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -589,7 +589,8 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                 const groupedCollection = groupDisplays(options.collection, options.groupHeaders);
                 this.collection = new Collection(groupedCollection);
             }
-            sessionStorage.submitDisabled = sessionStorage.submitDisabled ?? true;
+            sessionStorage.submitDisabled = sessionStorage.submitDisabled === undefined ?
+                true: sessionStorage.submitDisabled;;
```

Comment:
Did the linter not like the `??`?
Also I think this condition might be what was disabling the submit button by default on staging. It seems like the condition would want to default to `false` unless the intent is to default to disabling the button.